action.yml

```diff
@@ -3,7 +3,7 @@ description: "Squads v4 program upgrade Github Action"
 author: "Valentin Madrid"
 branding:
   icon: "upload"
-  color: "black"
+  color: "gray-dark"
```

Comment:
Apparently `black` is not a valid value.
https://json.schemastore.org/github-action.json